cmake/ConfigDefault.cmake

```
@@ -52,12 +52,12 @@ set (GMT_PACKAGE_VERSION_MAJOR 6)
 set (GMT_PACKAGE_VERSION_MINOR 2)
 set (GMT_PACKAGE_VERSION_PATCH 0)
 # If this is a beta version or similar, add a string suffix
-# set (GMT_PACKAGE_VERSION_SUFFIX "")
+set (GMT_PACKAGE_VERSION_SUFFIX "-rc1")
```

I think we should set it to `rc1`, without the hyphen, so that the package name is `gmt-6.2.0rc1-xxx`, consistent with our old rc releases.
```suggestion
set (GMT_PACKAGE_VERSION_SUFFIX "rc1")
```

---

Done